src/mono/netcore/sample/Android/Program.csproj

```diff
@@ -38,12 +38,12 @@
     <Copy SourceFiles="@(BclBinaries)" DestinationFolder="$(BundleDir)\%(RecursiveDir)" SkipUnchangedFiles="true"/>
 
     <!-- TEMP: consume OpenSSL binaries from external sources via env. variables -->
-    <Copy Condition="'$(AndroidOpenSslCryptoLib)' != ''"
-          SourceFiles="$(AndroidOpenSslCryptoLib)"
-          DestinationFolder="$(BundleDir)" SkipUnchangedFiles="true"/>
-    <Copy Condition="'$(AndroidOpenSslLib)' != ''"
-          SourceFiles="$(AndroidOpenSslLib)"
-          DestinationFolder="$(BundleDir)" SkipUnchangedFiles="true"/>
+    <Copy Condition="'$(AndroidOpenSslAar)' != ''"
+          SourceFiles="$(AndroidOpenSslAar)\prefab\modules\crypto\libs\android.$(AndroidAbi)\libcrypto.so"
+          DestinationFolder="$(PublishDir)" SkipUnchangedFiles="true"/>
```

Comment:
I don't think PublishDir will work here since we're not running publish in the sample.

Reply:
oops, copy-paste